### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by LucidityDesign**
- feat(Calendar): add standalone calendar component

### CODE REVIEW

File: src/components/controls/dates/Calendar/Calendar.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import { Calendar as MantineCalendar } from "@mantine/dates"
+import { useTheme } from "../../../../hooks/useTheme"
+import { Box, BoxProps } from "../../../layout/Box/Box"
+
+interface CalendarBaseProps {
+  /** Maximum possible date. */
+  maxDate?: Date
+
+  /** Minimum possible date. */
+  minDate?: Date
```
Comment (LucidityDesign) [inline src/components/controls/dates/Calendar/Calendar.tsx:11]: Not sure if we need `minDate`. We don't have it in DatePicker

File: src/components/controls/dates/Calendar/Calendar.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+  /** Called when date changes. */
+  onChange: (value: CalendarValue) => void
```
Comment (LucidityDesign) [inline src/components/controls/dates/Calendar/Calendar.tsx:14]: As the calendar is always controlled, I set `onChange` as required.

File: src/components/controls/dates/Calendar/Calendar.tsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+  /** Selected date, required with controlled input. */
+  value?: CalendarValue
+
+  /** Props passed to root element. */
+  wrapperProps?: BoxProps
+
+  /** Accessible name */
+  label: string
```
Comment (LucidityDesign) [inline src/components/controls/dates/Calendar/Calendar.tsx:23]: I'm not sure how you handle a11y. If the component is used with a descriptive headline we don't need a aria-label and I should change it to optional

File: src/components/controls/dates/Calendar/Calendar.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  const currentDayButton = canvas.getByRole("button", { name: "10" })
+  expect(currentDayButton.getAttribute("data-selected")).toBe("true")
+}
+
+export const Mouse = Template.bind({})
+Mouse.args = {
+  value: new Date(new Date().setDate(10)),
+}
+
+Mouse.play = async ({ canvasElement, args }) => {
```
Comment (filiptammergard) [inline src/components/controls/dates/Calendar/Calendar.stories.tsx:48]: I have sticked to grouping the template, args and interaction tests without empty lines between to make it easy to see the full "story". What do you think about doing that here as well?

```suggestion
export const Mouse = Template.bind({})
Mouse.args = {
  value: new Date(new Date().setDate(10)),
}
Mouse.play = async ({ canvasElement, args }) => {
```

**Review by filiptammergard — COMMENTED** [review]
Great! I think we can hold off merge here until tomorrow so that we can show it for Rui and ask for feedback.

File: src/components/controls/dates/Calendar/Calendar.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+import { Calendar } from "./Calendar"
+
+export default {
+  title: "Controls/Dates/Calendar",
+  component: Calendar,
+  argTypes: {
+    disabled: {
+      control: "boolean",
+    },
+  },
```
Comment (filiptammergard) [inline src/components/controls/dates/Calendar/Calendar.stories.tsx:14]: I think this should be removed, as we don't have a `disabled` prop, right?

File: src/components/controls/dates/Calendar/Calendar.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+} as ComponentMeta<typeof Calendar>
+
+const Template: ComponentStory<typeof Calendar> = (args) => {
+  const { value: defaultValue } = args
+  const [value, setValue] = useState<Date | null>(defaultValue || null)
```
Comment (filiptammergard) [inline src/components/controls/dates/Calendar/Calendar.stories.tsx:19]: What do you think about inferring types like this?

```suggestion
  const [value, setValue] = useState(defaultValue)
```

**Review by filiptammergard — APPROVED** [review]
Nice! When these comments are resolved I think this is ready to go 🙂

**Comment by LucidityDesign** [comment]
Thanks for the great Feedback

**Pull request merged**